trunk/SUAVE/Input_Output/VTK/save_fuselage_vtk.py

```diff
@@ -0,0 +1,127 @@
+## @ingroup Input_Output-VTK
+# save_fuselage_vtk.py
+# 
+# Created:    Jun 2021, R. Erhard
+# Modified: 
+#  
+
+from SUAVE.Plots.Geometry_Plots.plot_vehicle import generate_fuselage_points
+import numpy as np
+
+def save_fuselage_vtk(vehicle, filename, Results):
+    """
+    Saves a SUAVE fuselage object as a VTK in legacy format.
+
+    Inputs:
+       vehicle        Data structure of SUAVE vehicle                [Unitless] 
+       filename       Name of vtk file to save                       [String]  
+       Results        Data structure of wing and propeller results   [Unitless]
+       
+    Outputs:                                   
+       N/A
+
+    Properties Used:
+       N/A 
+    
+    Assumptions:
+       N/A
+    
+    Source:  
+       None
+    
+    """    
+    
+    fus_pts = generate_fuselage_points(vehicle.fuselages.fuselage)
+    num_fus_segs = np.shape(fus_pts)[0]
+    if num_fus_segs == 0:
+        print("No fuselage segments found!")
+    else:
+        write_fuselage_vtk(fus_pts,filename)
```

Comment:
I like splitting this function out. But name these functions very different things so the user doesn't call the wrong one.

Reply:
renamed to write_fuselage_data